frontend-react/src/components/Table/Table.test.tsx

```diff
@@ -269,7 +269,7 @@ describe("Table, filter integration tests", () => {
             defaultCursor
         );
         // Checking for inclusive date
-        expect(screen.getByText(/cursor:/)).toHaveTextContent(/23:59:59.000Z/);
```

Comment:
Is the difference here going to be an issue anywhere?

Reply:
It shouldn't be since this is what is being sent as the 'to' date in the filter to return the table results.